src/static/index.html

```diff
@@ -13,6 +13,8 @@
           rel="stylesheet" type="text/css">
     <link href="assets/css/markdown.css"
           rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+          rel="stylesheet">
```

Comment:
update: talked to Kristian, linking at runtime should be fine, no action needed.